src/Microsoft.Owin.Security.Authorization/IResourceAuthorizationHelper.cs

```diff
@@ -14,10 +14,11 @@ public interface IResourceAuthorizationHelper
         /// <param name="controller">The controller from which <see cref="AuthorizationOptions"/> may be obtained.</param>
         /// <param name="user">The user to evaluate the authorize data against.</param>
         /// <param name="authorizeAttribute">The <see cref="IAuthorizeData"/> to evaluate.</param>
+        /// <param name="resource">The resource to evaluate the authorize data against.</param>
         /// <returns>
         /// A flag indicating whether authorization has succeeded.
         /// This value is <value>true</value> when the <paramref name="user"/> fulfills the <paramref name="authorizeAttribute"/>; otherwise <value>false</value>.
         /// </returns>
-        Task<bool> IsAuthorizedAsync(IAuthorizationController controller, ClaimsPrincipal user, IAuthorizeData authorizeAttribute);
+        Task<bool> IsAuthorizedAsync(IAuthorizationController controller, ClaimsPrincipal user, IAuthorizeData authorizeAttribute, object resource = null);
```

Comment:
From the build logs:
##[warning]Microsoft.Design : Replace method 'IResourceAuthorizationHelper.IsAuthorizedAsync(IAuthorizationController, ClaimsPrincipal, IAuthorizeData, object)' with an overload that supplies all default arguments.

Reply:
Sorry for the delay,

I have fixed it, I guess I screwed up during my initial commit. Thank you for your patience.